helm/values.yaml

```diff
@@ -33,7 +33,7 @@ operator:
 # value of notification-manager
 notificationmanager:
   image:
-    tag: latest
+    tag: v0.6.0
     pullPolicy: Always
```

Comment:
Always => IfNotPresent ?

Reply:
It will not pull image when image update when the tag is `latest` and pullPolicy is `IfNotPresent`